README.md

```diff
@@ -1 +1,41 @@
-# Capstone-1
\ No newline at end of file
+
```

Comment:
- [ ] @Wahaj-Ali as the previous reviewer indicated kindly change your readme file to the latest one, here is the link to [the template](https://github.com/microverseinc/readme-template)